Adds installation page for IPFS and Docker. (#795)
ipfs/ipfs-docs: docs/how-to/run-ipfs-inside-docker.md

---
title: Run IPFS inside Docker
description: You can run IPFS inside Docker to simplify your deployment processes, and horizontally scale your IPFS infrastructure.
---

# Run IPFS inside Docker 

You can run IPFS inside Docker to simplify your deployment processes, as well as horizontally scale your IPFS infrastructure.

> 1. Grab the IPFS docker image hosted at [hub.docker.com/r/ipfs/go-ipfs](https://hub.docker.com/r/ipfs/go-ipfs/).

Author: ```suggestion
## Set up

1. Grab the IPFS docker image hosted at [hub.docker.com/r/ipfs/go-ipfs](https://hub.docker.com/r/ipfs/go-ipfs/).
```

---
title: Run IPFS inside Docker

> description: You can run IPFS inside Docker to simplify your deployment processes, as well as horizontally scale your IPFS infrastructure.

Author: ```suggestion
description: You can run IPFS inside Docker to simplify your deployment processes, and horizontally scale your IPFS infrastructure.
```
